framework/audio/engine/iaudioengine.h

```diff
@@ -38,9 +39,13 @@ class IAudioEngine : MODULE_EXPORT_INTERFACE
     using handle = unsigned int;
     using time = float;
 
+    virtual Ret init() = 0;
+    virtual void deinit() = 0;
+    virtual bool isInited() const = 0;
+
     virtual float sampleRate() const = 0;
 
-    virtual handle play(IAudioSource* s, float volume = -1, float pan = 0, bool paused = false) = 0;
+    virtual handle play(std::shared_ptr<IAudioSource> s, float volume = -1, float pan = 0, bool paused = false) = 0;
```

Comment:
Change to source, pls

Reply:
Renamed to "src", if rename to "source", the line of code will be more than 120, and I will have to wrap to another line.
At the same time, in this case, there is little value in the full name, because the data type uniquely determines what needs to be a pass.